db/migrations/20220605211859_add_full_text_search_with_pgsql/migration.sql

```diff
@@ -48,11 +48,11 @@ CREATE OR REPLACE FUNCTION project_members_versions_fn() RETURNS TRIGGER
   LANGUAGE plpgsql AS $body$
 BEGIN
   IF (TG_OP = 'INSERT') THEN
-    INSERT INTO "ProjectMembersVersions"("projectId", "profileId", "hoursPerWeek", "active", "practicedSkills")
-    VALUES (new."projectId", new."profileId", new."hoursPerWeek", new.active, '');
+    INSERT INTO "ProjectMembersVersions"("projectId", "profileId", "hoursPerWeek", "role", "active", "practicedSkills")
+    VALUES (new."projectId", new."profileId", new."hoursPerWeek", new.role, new.active, '');
   ELSIF (TG_OP = 'UPDATE') THEN
-    INSERT INTO "ProjectMembersVersions"("projectId", "profileId", "hoursPerWeek", "active", "practicedSkills")
-    VALUES (new."projectId", new."profileId", new."hoursPerWeek", new.active, '');
+    INSERT INTO "ProjectMembersVersions"("projectId", "profileId", "hoursPerWeek", "role", "active", "practicedSkills")
+    VALUES (new."projectId", new."profileId", new."hoursPerWeek", new.role, new.active, '');
```

Comment:
This files cannot be modified, since this migration already ran, production will ignore them. Instead, if you want to modify this triggers, you need to create a new migration file, and drop and recreate the trigger.

Reply:
Yep... I noticed that, so I re-rolled these changes in the last commit, right now the file is just like the original.

Thanks for the advice!